### ISSUE TITLE
Show function of peak-shaving application

### ISSUE THREAD

**Issue by sfeilmeier** [OpenEMS/openems#11]
From openems-gui created by [sfeilmeier](https://github.com/sfeilmeier) : OpenEMS/openems-gui#30  

### Bug Report or Feature Request (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [x] feature request
```

### Desired functionality.

There needs to be a component that shows the (historic and current) function of a peak-shaving application.

**Comment by sfeilmeier** [comment]
Here a draft for monitoring of peak-shaving application:

1st graph: Loads and defined limitation
2nd graph: Battery charge/discharge
3rd graph: External load dumping (switch off loads)
4th graph: Grid meter

![grafik](https://user-images.githubusercontent.com/3515268/27782473-fa722e58-5fd2-11e7-8e18-543a34963ebf.png)

**Comment by sfeilmeier** [comment]
Fixed in  #1048 and #1057
